public/app/features/alerting/components/AlertDefinitionItem.tsx

```diff
@@ -21,6 +21,12 @@ export const AlertDefinitionItem: FC<Props> = ({ alertDefinition, search }) => {
           <span key="text">{alertDefinition.description}</span>
         </span>
       </Card.Meta>
+      <Card.Actions>
+        <div />
```

Comment:
Is this empty div necessary?

Reply:
Well, the `<Card.Actions>` components expects an array of children. So it's either changing the Actions or having the dummy div or wrapping the LinkButton in an array.